### DISCUSSION TITLE
[GUI] cant add pool url

### DISCUSSION THREAD

**Discussion by killtux** [Chia-Network/chia-blockchain#9072] [Support]
cant add pool since i cant type oder copy in the "Connect with Pool: Pool URL" Field.
The XCH field below works but not the URL field above:

![image](https://user-images.githubusercontent.com/36768145/139710181-71ea5ddc-5705-42c5-b5f3-4a31dfca2682.png)

Am i too dump or is this really a bug? i am running Win10 x64 and newest version of chia

**Comment by Faysk** [comment]
I have the same problem =(
![image](https://user-images.githubusercontent.com/5545917/139774014-fc3c381a-8615-414b-9489-9688c2529b84.png)
![image](https://user-images.githubusercontent.com/5545917/139774287-c6af64a6-d65f-4794-ba05-5cac1bdb76fc.png)

Windows 10 Pro Edition
Version 21H1
Installed on ‎27/‎08/2021
Compilation of SO 19043.1320
Windows Experience Feature Experience Pack 120.2212.3920.0

**Reply by Faysk** [reply]
For now I'm using CLI in powershell: 

###Command to access chia folder:
Set-Location $ENV:LOCALAPPDATA\chia-blockchain\app*\resources\app.asar.unpacked\daemon

###Command to show nft join options
.\chia plotnft join -h

###Command to show nft wallets id:
.\chia plotnft show

###Command to enter the pool:
.\chia plotnft join -i 7 -u https://pool.fazendachia.com.br  -y

**Reply by ChrystianDuarte** [reply]
hi @Faysk  i try to join but i have a error with the get_current_start of wallet. I believe that the migration between 1.2.7 to 1.2.10 is not support. i have to add a issue to the project.

`{'authentication_token_timeout': 5,
 'description': 'Welcome to Space Pool. By creating this Plot NFT, you agree '
                'to the following terms https://pool.space/terms',
 'fee': 0.01,
 'logo_url': 'https://pool.space/logo-250.png',
 'minimum_difficulty': 1,
 'name': 'Space Pool',
 'protocol_version': 1,
 'relative_lock_height': 64,
 'target_puzzle_hash': '0x934293842-------xxxxxx'}

Will join pool: https://na1.pool.space with Plot NFT xxxxxxx.
Error performing operation on Plot NFT -f 1638xxxxx wallet id: 1: {'error': "'Wallet' object has no attribute 'get_current_state'", 'success': False}
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x0000xxxxxxx>
Traceback (most recent call last):
  File "asyncio\proactor_events.py", line 116, in __del__
  File "asyncio\proactor_events.py", line 108, in close
  File "asyncio\base_events.py", line 746, in call_soon
  File "asyncio\base_events.py", line 510, in _check_closed
RuntimeError: Event loop is closed`

**Reply by Faysk** [reply]
You are probably using your normal wallet id (-i 1) and you need to use the pooling wallet id - typically this is id 2.

.\chia plotnft show

.\chia plotnft join -i 2 -u https://na1.pool.space -y

**Comment by ChrystianDuarte** [comment]
I have  the same problem 
![4DD23D2F-4B9F-4C85-B2FC-9F0AAC13B866](https://user-images.githubusercontent.com/19255861/139867643-01afe321-756f-4cca-bc34-2b6bf83cd5c0.jpeg)

Windows 10 Pro Edition
Installed on ‎01/‎09/2021
Migrate - 1.2.7 to 1.2.10
